Cartfile.resolved

```diff
@@ -1,13 +1,13 @@
-github "Alamofire/Alamofire" "9faceeafaa675ec0a4e2c8001a641157844b9973"
-github "norio-nomura/Base32" "v0.3.0"
-github "wesj/Deferred" "ef8e95954196c815aff79f0b4471219ab2885692"
+github "Alamofire/Alamofire" "2f94c11bb039bd7e3e43b925e20c8e536c9be757"
+github "mozilla/Base32" "d2136486487a77336ae18b1bd2f9a3f689ec44e5"
```

Comment:
Do we want to use mozilla as the owner of these forks? I feel like if we make mozilla the owner it makes it seem that we're going to be maintaining our own fork instead of temporarily using a fork for swift 2.0 changes.